habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -1680,7 +1734,21 @@ def _step_rel_waypoint(
                 xyt
             )
 
+            # Query velocity controller for control input of pitch of camera
+            camera_pitch_angular_err = (
+                goal_camera_pitch_ang - self.camera_pitch_ang
+            )
+            camera_pitch_angular_velocity = self.w2v_controller.controller.control._velocity_feedback_control(
+                camera_pitch_angular_err,
+                self._acc_camera_pitch_ang,
+                self._w_camera_pitch_max,
+            )
```

Comment:
Even though all members are made default in Python, it's generally not advised to access nested members like this due to ease of debug.
We have a `ContinuousController` wrapper class as the interface to the controllers. It would be cleaner to implement a method there to expose velocity feedback control for camera tilt use.